src/oci/runc.rs

```diff
@@ -0,0 +1,398 @@
+//! Interface to [runc][0], a OCI compliant cli for spawning and running containers.
```

Comment:
could this be called just OCI runtime?

From what I can see below, there is nothing specific to runc, so other runtimes can also be used

Reply:
I will update it. I think you are correct. Thank you!

Reply:
done.